src/window/linux.cc

```diff
@@ -4,6 +4,9 @@ static GtkTargetEntry droppableTypes[] = {
   { (char*) "text/uri-list", 0, 0 }
 };
 
+#define DEFAULT_MONITOR_WIDTH 720
+#define DEFAULT_MONITOR_HEIGHT 364
```

Comment:
actually we should take these values from `windowManager->options.default{Width|height}`

Reply:
That won't work if `[window] height = 50%` is set

Reply:
What was the issue with percents? I don't remember. all values should be in pixels in the end